src/chrome/chromeDebugSession.ts

```diff
@@ -33,6 +33,21 @@ export interface IChromeDebugSessionOpts extends IChromeDebugAdapterOpts {
     extensionName: string;
 }
 
+export type ExceptionType = "uncaughtException" | "unhandledRejection" | "generic";
+
+export interface  ITelemetryPropertiesPayload {
```

Comment:
Can this interface name be more specific? This isn't for all telemetry messages, just timings and errors.

Reply:
Renamed